opm/common/OpmLog/Logger.cpp

```diff
@@ -38,6 +38,18 @@ namespace Opm {
         addMessageType( Log::MessageType::Bug , "bug");
     }
 
+    void Logger::addTaggedMessage(int64_t messageType, const std::string& tag, const std::string& message) const {
+        if ((m_enabledTypes & messageType) == 0)
+            throw std::invalid_argument("Tried to issue message with unrecognized message ID");
+
+        if (m_globalMask & messageType) {
+            for (auto iter = m_backends.begin(); iter != m_backends.end(); ++iter) {
```

Comment:
Is it necessary to have two essentially identical `Logger::addMessage()` methods - couldn't the no-tag-version just call the with-tag-version with tag ""?